File: TuneINN/app/src/main/java/com/idlikadai/tuneinn/MainActivity.java
package com.idlikadai.tuneinn;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    public static boolean isFirstTime = true;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView textView = (TextView) findViewById(R.id.TextView_title);
        textView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent =new Intent(MainActivity.this,ListActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: ButtonCounter/app/src/main/java/com/idlikadai/buttoncounter/MainActivity.kt
package com.idlikadai.buttoncounter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val textCounter = findViewById<TextView>(R.id.textCounter);
        val buttonCounter = findViewById<Button>(R.id.buttonCounter);
        var counter = 0
        buttonCounter.setOnClickListener {
            counter+=1
            textCounter.text = counter.toString()
        }

        // Reset the count value
        buttonCounter.setOnLongClickListener {
            Toast.makeText(this, "Counter reset",Toast.LENGTH_SHORT).show();
            counter = 0;
            textCounter.text = 0.toString();
            true
        }
    }
}